README.md

```diff
@@ -128,9 +132,9 @@ jobs:
 | `retry_count`               | <span style="color:green"> optional </span>  | If a request fails, retry this number of times. Defaults to 1    |
 | `save`                      | <span style="color:green"> optional </span>  | A path to a csv file to save results to                          |
 | `timeout`                   | <span style="color:green"> optional </span>  | The timeout to provide to requests to wait for a response.       |
-| `white_listed_urls`         | <span style="color:green"> optional </span>  | A comma separated links to exclude during URL checks.            |
-| `white_listed_patterns`     | <span style="color:green"> optional </span>  | A comma separated patterns to exclude during URL checks.         |
-| `white_listed_files`        | <span style="color:green"> optional </span>  | Full paths to files to exclude (comma separated list).           |
+| `exclude_urls`              | <span style="color:green"> optional </span>  | A comma separated links to exclude during URL checks.            |
+| `exclude_patterns`          | <span style="color:green"> optional </span>  | A comma separated patterns to exclude during URL checks.         |
+| `exclude_files`             | <span style="color:green"> optional </span>  | Full paths to files to exclude (comma separated list).           |
```

Comment:
maybe: 
- A comma separated list of links
- A comma separated list of patterns
or 
- Links to exclude (comma separated)
- Patterns to exclude (comma separated)

Reply:
Sure! In the future you can just make suggestions for each line that I can directly commit.